code.gs

```diff
@@ -1659,6 +1659,38 @@ var i18n = {
     'It looks like you are using an outdated version of this script': 'Parece que tens uma versão desatualizada deste script',
     'You can find the latest one here': 'Podes encontrar a última versão aqui',
   },
+  'th': {
+    'Age': 'อายุ',
+    'Years': 'ปี',
```

Comment:
I've run all these strings through Google Translate to prevent any mismatch or big misundertanding which can easily happen when translating strings out of context.  
It all looks OK to me, except that most of the names (e.g. 'years', 'anniversaries'. 'birthdays') are translated as singular isntead of plural (e.g. 'year', 'anniversary', 'birthday'). Could you please confirm that this is a problem with Google Translate (which is not exactly super-reliable, I know) and the names are translated correctly as plurals?

Reply:
The Thai language doesn't have singular or plural. Hence, the names mean the same thing :)

Reply:
Perfect, thanks!